cmd/influxd/config.go

```diff
@@ -109,6 +109,32 @@ type Config struct {
 	Logging struct {
 		File string `toml:"file"`
 	} `toml:"logging"`
+
+	ContinuousQuery struct {
+		// when continuous queries are run we'll automatically recompute previous intervals
+		// in case lagged data came in. Set to zero if you never have lagged data. We do
+		// it this way because invalidating previously computed intervals would be insanely hard
+		// and expensive.
+		RecomputePreviousN int `toml:"recompute_previous_n"`
```

Comment:
The style in the rest of the config file (as far as I can see) is that `-` is used as the word delimiter for keys within a config section.